### ISSUE TITLE
Breadcrumbs: A11y Issue - The role of current page text should be 'text'

### ISSUE THREAD

**Issue by sleric1024** [SAP/ui5-webcomponents-react#3418]
**Describe the bug**
When reading "current page" (**Notebooks**) its announced as a "link" while its not a link but just plain text

**Isolated Example**
https://codesandbox.io/s/competent-smoke-mqee3x?file=/src/App.js

**To Reproduce**
1. Go to https://mqee3x.csb.app/
2. Inspect with Chrome developer tool
3. Check the text **Notebooks**
4. The role is marked as 'link' but it's just a plain text

**Expected behavior**
The role should be text

**Screenshots**
![image](https://user-images.githubusercontent.com/5724592/186556713-ae68b0ac-c9b0-4077-873f-82cf7e9c7df4.png)

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: 1.6.0
`@ui5/webcomponents-react` version: 0.27.0
Operating System: MacOS
Browser: Chrome

**Additional context**
I am from SAP Successfactors, and this issue is reported from SF a11y expert.

**Comment by Lukas742** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.
